guide/faq.md

```diff
@@ -43,7 +41,7 @@ However, if you are using the ExpandCollpase component, we recommend the use of
 
 The majority of TDS styles are component styles that have been generated by our build tooling to prevent conflicts with application styles. However, there are a small amount of page level styles to establish a common baseline.
 
-Specifically, the `tds.css` stylesheet contains the following page level styles:
+Specifically, the `@tds/core-css-reset/index.css` stylesheet contains the following page level styles:
```

Comment:
Maybe instead we just link to the CSS Reset package on npm or in the component catalogue and have this information there?

Reply:
I'll move this info over to the core-css-reset documentation